src/language-server/simple-ui-validator.ts

```diff
@@ -82,4 +83,12 @@ export class SimpleUiValidator {
             })
         }
     }
+    checkHeadingLevel(el: Heading, accept: ValidationAcceptor): void {
+        if (el.level > 6 || el.level < 1) {
+            accept('error', `Heading level can't be ${el.level}.`, { node: el, property: 'level' })
```

Comment:
What about changing the error message to "Heading level must be between 1 and 6"?

Reply:
We can do that but I'd like to atleast output the current (wrong) value so I'm gonna see how I can combine the two variants.
Maybe `Wrong headinglevel ${el.level}. Expected value between 1 and 6.` will do?

Reply:
Sounds good